README.md

```diff
@@ -84,7 +84,7 @@ Memory usage:   85.1M out of 985.4M
 
 ```
 $ multipass shell dancing-chipmunk
-Welcome to Ubuntu 18.04.1 LTS (GNU/Linux 4.15.0-42-generic x86_64)
+Welcome to Ubuntu 20.04.1 LTS (GNU/Linux 4.15.0-42-generic x86_64)
```

Comment:
Don't forget to change the output of `lsb_release -a` below. Notably, the codename should change from `bionic` to `noble`.